MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -194,7 +194,7 @@ public override bool RequiresRefresh()
         /// </summary>
         /// <param name="item">The item.</param>
         /// <exception cref="InvalidOperationException">Unable to add  + item.Name.</exception>
-        public void AddChild(BaseItem item)
+        public virtual void AddChild(BaseItem item)
```

Comment:
big change for a patch release

Reply:
I can revert it and put the logic back into the service if you prefer. This is cleaner though.